src/test/FileTest/FileControllerIntegrationTests.java

```diff
@@ -0,0 +1,54 @@
+package FileTest;
+
+import Security.EncryptionUtils;
+import TestUtils.TestUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+import java.nio.file.Paths;
+
+import static FileTest.FileControllerIntegrationTestHelperMethods.uploadTestPDF;
+
+public class FileControllerIntegrationTests {
+  private static EncryptionUtils encryptionUtils;
+  public static String username = "adminBSM";
+  ;
+
+  public static String currentPDFFolderPath =
+      Paths.get("").toAbsolutePath().toString()
+          + File.separator
+          + "src"
+          + File.separator
+          + "test"
+          + File.separator
+          + "PDFTest";
+
+  public static String resourcesFolderPath =
+      Paths.get("").toAbsolutePath().toString()
+          + File.separator
+          + "src"
+          + File.separator
+          + "test"
+          + File.separator
+          + "resources";
+
+  @BeforeClass
+  public static void setUp() {
+    TestUtils.startServer();
+    TestUtils.setUpTestDB();
```

Comment:
i dont think you need to set up the test db here btw- lets try to use the entity factory which allows us to start from a blank slate